Makefile

```diff
@@ -44,28 +45,33 @@ install:
 
 build: $(VENV_PYTHON) install-elcapitan install
 
-clean-env:
-	@cp molotov.env molotov.env.OLD
-	@rm -f molotov.env
-	@touch molotov.env
-
-configure: build
-	@bash loads-broker.tpl
 
 test: build
 	bash -c "URL_SERVER=$(URL_SERVER) $(BIN)/molotov -d $(TEST_DURATION) -cx loadtest.py"
 
 test-heavy: build
 	bash -c "URL_SERVER=$(URL_SERVER) $(BIN)/molotov -p $(TEST_PROCESSES_HEAVY) -d $(TEST_DURATION_HEAVY) -w $(TEST_CONNECTIONS_HEAVY) -cx loadtest.py"
 
+
 docker-build:
 	docker build -t firefoxtesteng/$(PROJECT)-loadtests .
 
 docker-run:
-	bash -c "docker run -e URL_SERVER=$(URL_SERVER) -e TEST_PROCESSES=$(TEST_PROCESSES) -e TEST_DURATION=$(TEST_DURATION) -e TEST_CONNECTIONS=$(TEST_CONNECTIONS) $(PROJECT)"
+	bash -c "docker run -e URL_SERVER=$(URL_SERVER) -e TEST_PROCESSES=$(TEST_PROCESSES) -e TEST_DURATION=$(TEST_DURATION) -e TEST_CONNECTIONS=$(TEST_CONNECTIONS) firefoxtesteng/$(PROJECT)-loadtests"
 
 docker-export:
 	docker save "$(PROJECT)/loadtest:latest" | bzip2> "$(PROJECT)-latest.tar.bz2"
 
+
+loads-config:
+	@bash loads-broker.tpl
+
+
 clean: 
 	@rm -fr venv/ __pycache__/ 
+
+clean-env:
+	@cp molotov.env molotov.env.OLD
```

Comment:
someone may want to try a clean set of env vars, but keep the original (default) around.  i added this because i'd clobbed my env file one too many times